sdk/nodejs/Makefile

```diff
@@ -47,9 +47,8 @@ istanbul_tests::
 	./node_modules/.bin/istanbul report text
 
 sxs_tests::
-# Intentionally disabled as PR https://github.com/pulumi/pulumi/pull/2609 breaks SxS
-# will be renabled once that goes in.
-#	pushd tests/sxs && yarn && tsc && popd
+	pushd tests/sxs_ts_3.6 && yarn && tsc && popd
+	pushd tests/sxs_ts_latest && yarn && tsc && popd
```

Comment:
updated to actually test different versions of ts.  I don't like that this is done with copy/pasting the test.  ideally we could truly have a matrix here where we test a whole bunch of combinations.  But i think this approach is acceptable for now given the time and amount of work we have.